plugins/plugin-optimize/README.md

```diff
@@ -0,0 +1,42 @@
+# @snowpack/plugin-optimize
+
+Optimize your unbundled Snowpack app:
+
+- ✅ Minify JS
+- ✅ [Preload JS Modules][modulepreload]
+- (Coming soon) Minify HTML
+- (Coming soon) Minify CSS
+
+### Usage
+
+From a terminal, run the following:
+
+```
+npm install --save-dev @snowpack/plugin-optimize
+```
+
+Then add this plugin to your Snowpack config:
+
+```js
+// snowpack.config.json
+{
+  "plugins": [
+    [
+      "@snowpack/plugin-optimize",
+      {
+        minifyJS: true, // default
+        preloadModules: true, // default
+      }
+    ]
+  ]
```

Comment:
By having this be an external plugin till this stabilizes, we can let the options bake a little bit.